templates/web/base/reports/_status_filter_options.html

```diff
@@ -0,0 +1,9 @@
+    [%~ IF c.cobrand.on_map_default_status == 'open' %]
+      data-none="[% loc('Open') %]"
+    [%~ END ~%]
+    data-all="[% loc('All') %]"
+    [%~ IF c.user.is_superuser OR c.user.from_body == body ~%]
```

Comment:
The conditional used to print out these options is `[% IF c.user_exists AND c.user.is_superuser OR c.user.belongs_to_body(body.id) %]` so it'd be good to match.